tests/qunit/tests/modules/dev-tools/assets/js/editor/dev-tools.spec.js

```diff
@@ -9,8 +15,51 @@ QUnit.module( 'File: modules/dev-tools/assets/js/editor/dev-tools.js', () => {
 			assert.deepEqual( { test: [ 'version', 'replacement' ] }, elementor.config.dev_tools.deprecation.soft_notices );
 		};
 
-		elementor.devTools.notifyDeprecated();
+		elementor.devTools.notifyBackendDeprecations();
 
 		elementorCommon.helpers.softDeprecated = softDeprecatedOrig;
 	} );
+
+	QUnit.test( 'parseVersion()', ( assert ) => {
+		// Act.
+		const version = elementor.devTools.parseVersion( '1.2.3.beta' );
+
+		// Assert.
+		assert.deepEqual( version, {
+			build: 'beta',
+			major1: 1,
+			major2: 2,
+			minor: 3,
+		} );
+	} );
+
+	QUnit.test( 'compareVersion()', ( assert ) => {
+		// Act.
+		const result = elementor.devTools.compareVersion( '1.2.3.beta', '1.2.3.beta' );
```

Comment:
what about greater and smaller versions?

Reply:
Handled